app/controllers/admin/other_case_types_report_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Admin::OtherCaseTypesReportController < AdminController
+  def index
+    @cases_with_other_type = TribunalCase.with_other_case_type.order(created_at: :desc)
+  end
+
+  private
+
+  # We are reusing the same credentials we created for the upload problems report, for simplicity.
```

Comment:
Should the `credentials` method be pushed up into the AdminController class? 

That would remove the duplication in the code, and also make it more obvious that we are using the same credentials throughout.